js/mexc3.js

```diff
@@ -0,0 +1,3659 @@
+            //         "3.550717",
+            //         1640804940000,
+            //         "168387.3"
+            //       ],
+            //     ]
+            //
+            candles = response;
+        } else if (market['swap']) {
+            if (since !== undefined) {
+                request['start'] = parseInt (since / 1000);
+            }
+            const priceType = this.safeString (params, 'price', 'default');
+            params = this.omit (params, 'price');
+            const method = this.getSupportedMapping (priceType, {
+                'default': 'contractPublicGetKlineSymbol',
+                'index': 'contractPublicGetKlineIndexPriceSymbol',
+                'mark': 'contractPublicGetKlineFairPriceSymbol',
+            });
+            const response = await this[method] (this.extend (request, params));
+            //
+            //     {
+            //         "success":true,
+            //         "code":0,
+            //         "data":{
+            //             "time":[1634052300,1634052360,1634052420],
+            //             "open":[3492.2,3491.3,3495.65],
+            //             "close":[3491.3,3495.65,3495.2],
+            //             "high":[3495.85,3496.55,3499.4],
+            //             "low":[3491.15,3490.9,3494.2],
+            //             "vol":[1740.0,351.0,314.0],
+            //             "amount":[60793.623,12260.4885,10983.1375],
+            //         }
+            //     }
+            //
+            const data = this.safeValue (response, 'data');
+            candles = this.convertTradingViewToOHLCV (data, 'time', 'open', 'high', 'low', 'close', 'vol');
+        }
+        return this.parseOHLCVs (candles, market, timeframe, since, limit);
+    }
+
+    async fetchIndexOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        return await this.fetchOHLCV (symbol, timeframe, since, limit, this.extend ({ 'price': 'index' }, params));
+    }
+
+    async fetchMarkOHLCV (symbol, timeframe = '1m', since = undefined, limit = undefined, params = {}) {
+        return await this.fetchOHLCV (symbol, timeframe, since, limit, this.extend ({ 'price': 'mark' }, params));
+    }
+
+    parseOHLCV (ohlcv, market = undefined) {
+        return [
+            this.safeInteger (ohlcv, 0),
+            this.safeNumber (ohlcv, 1),
+            this.safeNumber (ohlcv, 2),
+            this.safeNumber (ohlcv, 3),
+            this.safeNumber (ohlcv, 4),
+            this.safeNumber (ohlcv, 5),
+        ];
+    }
+
+    async fetchTickers (symbols = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        if (symbols !== undefined) {
+            request['symbol'] = symbols.join (',');
+        }
+        const [ marketType, query ] = this.handleMarketTypeAndParams ('fetchTickers', undefined, params);
+        let tickers = undefined;
+        if (marketType === 'spot') {
```

Comment:
After some tests and looking into the docs, I don't think this "," separation in the URL is valid (see images below). To fetch multiple tickers you omit the symbol so the exchange returns everything and you do the filtering afterward.



![image](https://user-images.githubusercontent.com/43336371/164719618-48fc7a1e-6e8b-4885-8ec0-d5afc2ea0361.png)

![image](https://user-images.githubusercontent.com/43336371/164719454-be8fe139-fb25-4095-93c6-ecfefcca90a2.png)

Reply:
yes right, as noted above, the request was incorrect from the beggining, i don't remind the reason, probably missed when integrated swaps.